sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java

```diff
@@ -33,6 +35,8 @@
     private TestEnvironment() {
         this.testMode = readTestModeFromEnvironment();
         this.serviceVersion = readServiceVersionFromEnvironment();
+        this.httpClientType = readHttpClientTypeFromEnvironment();
+        System.out.println(String.format("Tests will run with %s http client", this.httpClientType));
```

Comment:
Can remove this once the PR is ready for merging

Reply:
I left this on purpose (like other printouts in this class are).